packages/gatsby-theme-apollo-core/gatsby-config.js

```diff
@@ -9,7 +9,9 @@ module.exports = {
     {
       resolve: 'gatsby-plugin-less',
       options: {
-        modifyVars: mapKeys(colors, (value, key) => `color-${key}`)
+        lessOptions: {
+          modifyVars: mapKeys(colors, (value, key) => `color-${key}`)
+        }
```

Comment:
The update for this plugin moves options to more specific [`lessOptions` and `loaderOptions`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-less#how-to-use)